blogs/nlp/LOREN/README.md

```diff
@@ -0,0 +1,135 @@
+
+Local premises are the collection of information to verify the veracity of each claim. [2] To construct local premises, we need probing questions and evidence as input. Regarding the probing questions, they consist of two sub-questions: one is the cloze question and another is the interrogative question. The cloze question is generated by masking out the phrase in the original claim and the interrogative question is generated by the off-the-shelf question generating model based on the T5 base. For the evidence retrieval part, LOREN first identifies the name entity in the claim, and then it searches for the relevant information from the Wikipedia(Since they used Wikipedia as the evidence retrieval source, it results in the lack of common sense Knowledge of LOREN, which I will discuss in detail in the third session).
+
+With probing questions and evidence, LOREN adopted the generative machine reading comprehension(MRC) model as a sequence-to-sequence model [3]. MRC model is self-supervised and it will generate answers for each phrase and replace the original phrase in the claim to generate our local premises.
+
+### Latent model construction
+
+Before we dive into the detail of latent model, let me introduce the logic constraints that LOREN developed for predicting claim veracity based on the veracity of each phrase.
+
+1. A claim is SUP iff all phrases are SUP
+2. A claim is REF iff there's at least one phrase that is REF
+3. A claim is NEI iff neither above.
+
+Following the above logic rules, loren further soften these rules into a probability distribution.
+
+1. $q^{T}_{\phi}(y_z = SUP) = \sum_{i=1}^{|z|}q_{\phi}(z = SUP)$
+2. $q^{T}_{\phi}(y_z = REF) = 1- \sum_{i=1}^{|z|}(1-q_{\phi}(z = SUP))$
+3. $q^{T}_{\phi}(y_z = NEI) = 1-q^{T}_{\phi}(y_z = SUP)-q^{T}_{\phi}(y_z = REF)$
+
+where
+
+1. $y\in{SUP,REF,NEI}$, which is the veracity of a claim.
+2. $z_i\in{SUP,REF,NEI}$, which is the latent variable represents the veracity of a phrase.
+3. $y_z$, which is the logic aggregation of the latent variable $z$, meaning that $y_z$ is a categorical distribution over three labels ${SUP,REF,NEI}$.
+
+The above aggregation of the soft logic serves to distill the knowldge to $p_{\theta}(y|z,x)$ (Probability of the claim label given the phrase-level label and input x),
+which is the probability we eventually want to optimize.
+
+## Experiment
+
+The model is trained on BERT and RoBERTa, and eventually it is evaluated on the dataset FEVER 1.0 shared task that is split into training, development and blind test set.
+Before we analyze the experiment result, I will summarize the experiment metrics they used to evaluate the performance[2]:
+
+### Experiment metrics for the performance of the model:
+
+1. Label Accuracy (LA): The accuracy of predicted label for claim regardless of retrieved evidence.
+2. FEVER score (FEV): The accuracy of both predicted label and retrieved evidence.
+
+![image2](./result1.png)
```

Comment:
convert to bar chart for the test FEV score?

Reply:
I will do it, would you like me to have bar charts for each of them, or having a bar chart for the test FEV score along would be ok?

Reply:
one bar chart for the test FEV is fine.